docs/start/framework/react/hosting.md

```diff
@@ -159,6 +159,19 @@ export default defineConfig({
 })
 ```
 
+You should opt-in to the nightly release channel by updating your `package.json`:
+
+```text
+{
+  "devDependencies": {
+--    "nitropack": "^2.0.0"
+++    "nitropack": "npm:nitropack-nightly@latest"
```

Comment:
Isn't this package just called "nitro" now?
https://github.com/nitrojs/nitro/blob/v3/docs/1.guide/00.migration.md#nitropack-is-renamed-to-nitro